aten/src/ATen/native/quantized/cpu/embedding_packed_params.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/core/ivalue.h>
+
+struct EmbeddingPackedParamsBase : public torch::jit::CustomClassHolder {
```

Comment:
nit: suggest to use `EmbeddingBagPackedParamsBase` to be more clear

Reply:
I think this struct can be shared by embedding and embedding_bag operators since it only does weight packing.